pkgs/applications/version-management/git-and-tools/default.nix

```diff
@@ -8,7 +8,7 @@ let
   gitBase = lib.makeOverridable (import ./git) {
     inherit fetchurl stdenv curl openssl zlib expat perl python gettext gnugrep
       asciidoc xmlto docbook2x docbook_xsl docbook_xml_dtd_45 libxslt cpio tcl
-      tk makeWrapper subversionClient gzip;
+      tk makeWrapper subversionClient gzip libiconvOrNull;
```

Comment:
I understand #6199 in the sense that libiconv will replace the functionality of libiconvOrNull. Either this is merged first and libiconvOrNull occurrences will be fixed as part of #6199 or we do it the other way round.